src/shared/logger.ts

```diff
@@ -1,6 +1,9 @@
 import errorStackParser from 'error-stack-parser'
 import StackFrame from 'stackframe'
 import { RC_Config } from './shared-types'
+import { stdout } from 'process'
+
+stdout.on('error', () => {})
```

Comment:
this file is shared between renderer and main process. we can not use node dependencies in the renderer process.

Reply:
I guess you meant to edit this file instead: https://github.com/deltachat/deltachat-desktop/blob/master/src/main/log-handler.ts

but also then your pr description tells about a different solution, so maybe you also forgot to commit some files?

Reply:
okay I will put the error handler in that file, there are other better solutions for v18 but for v16 which Desktop is currently using, this is the best I accomplished. Also according to linkxt in the issue, on UNIX(-like), you have to try writing and see if you get error or not.

Reply:
we can also upgrade the node version to 18 in a dedicated pr: updating all docs, readme, github action ci files and so on.

Reply:
Not sure if upgrading to v18 and using `stdout.closed` will solve this crash issue.